services/user.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from db.models import User

```

Comment:
The import of `User` from `db.models` is redundant because the code uses `get_user_model()` to interact with the user model. Consider removing this unused import to clean up the code.